devnets/startNetwork.ts

```diff
@@ -17,6 +17,59 @@ export interface NetworkChain {
   ports: number[]
 }
 
+export async function startNetworkForMetadata(
+  tempDir: string,
+  config: NetworkConfig,
+  signal: AbortSignal,
+): Promise<Network> {
+  const relayBinary = await resolveBinary(config.binary, signal)
+  const relaySpec = await createCustomChainSpec(
+    path.join(tempDir, "relay"),
+    relayBinary,
+    config.chain,
+    () => {},
+  )
+  return {
+    relay: await spawnChain(
+      path.join(tempDir, "relay"),
+      relayBinary,
+      relaySpec,
+      1,
+      [],
+      relayBinary,
+      signal,
+    ),
+    paras: Object.fromEntries(
+      await Promise.all(
```

Comment:
Since the paras no longer need to be spawned after the relay, these two `await`ed functions should be done in parallel